Gordon360/Models/ViewModels/RecIM/Scheduler.cs

```diff
@@ -7,6 +7,7 @@ namespace Gordon360.Models.ViewModels.RecIM
     public class EliminationRound
     {
         public int TeamsInNextRound { get; set; }
+        public int NumByeTeams { get; set; }
```

Comment:
This could perhaps use a comment since it is a technical recim-specific (sports-specific) term. I didn't know it, so I assume others might have the same confusion.

Reply:
I'll make sure I leave a big chunk of logic documentation for the scheduling things after Double Elimination is done. I will make this a point worth noting!

Reply:
Cool thanks!